firebase-common/build.gradle.kts

```diff
@@ -46,15 +46,8 @@ kotlin {
         publishAllLibraryVariants()
     }
 
-    fun nativeTargetConfig(): KotlinNativeTarget.() -> Unit = {
-
-    }
-
-    if (project.extra["ideaActive"] as Boolean) {
-        iosX64("ios", nativeTargetConfig())
-    } else {
-        ios(configure = nativeTargetConfig())
-    }
+    ios()
```

Comment:
not sure about this, if you change it here, you should change it for all of the build gradles e.g. firebase-auth, etc

Reply:
can do, will do 👌

Reply:
✅